<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnclosureNewDialog</class>
 <widget class="QWidget" name="EnclosureNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="2" column="3">
    <widget class="QLineEdit" name="m_pQtsMin">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #606060;Border: 1px solid #606060; border-radius: 5px; color: #606060;;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="NW::Label" name="label_7">
     <property name="text">
      <string>SPL (min):</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QLineEdit" name="m_pSPLMax">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">Border: 1px solid #606060; border-radius: 5px; color: #606060;;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#606060</string>
     </property>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="NW::Label" name="label_5">
     <property name="text">
      <string>Qts (min):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLineEdit" name="m_pFsMax">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">Border: 1px solid #606060; border-radius: 5px; color: #606060;;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="NW::Label" name="label_2">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resource/sival.qrc">:/sival/filter_dark.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="NW::Label" name="label_17">
     <property name="text">
      <string>Hz</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="m_pSPLMin">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">Border: 1px solid #606060; border-radius: 5px; color: #606060;;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QLineEdit" name="m_pRMSMax">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">Border: 1px solid #606060; border-radius: 5px; color: #606060;;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="NW::Label" name="label_12">
     <property name="text">
      <string>RMS (max):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="NW::Label" name="label_3">
     <property name="text">
      <string>Fs (max):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="m_pFsMin">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">Border: 1px solid #606060; border-radius: 5px; color: #606060;;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="NW::Label" name="label_9">
     <property name="text">
      <string>Diameter:(inch)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="NW::Label" name="label_10">
     <property name="text">
      <string>Re(Ohm):</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="NW::Label" name="label_15">
     <property name="text">
      <string>dB</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="NW::Label" name="label_6">
     <property name="text">
      <string>Qts (max):</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="NW::Label" name="label_11">
     <property name="text">
      <string>SPL (max):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="8">
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="m_pManufacturer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>348</width>
        <height>380</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="HeaderLabel" name="label">
     <property name="text">
      <string>Enclosure Settings Filter</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QComboBox" name="comboBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#606060;</string>
     </property>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="NW::Label" name="label_20">
     <property name="text">
      <string>W</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="NW::Label" name="label_13">
     <property name="text">
      <string>Hz</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="NW::Label" name="label_4">
     <property name="text">
      <string>Fs (min):</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="m_pRMSMin">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">Border: 1px solid #606060; border-radius: 5px; color: #606060;;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="NW::Label" name="label_16">
     <property name="text">
      <string>W</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="NW::Label" name="label_19">
     <property name="text">
      <string>dB</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="NW::Label" name="label_8">
     <property name="text">
      <string>RMS (min):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLineEdit" name="m_pQtsMax">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">Border: 1px solid #606060; border-radius: 5px; color: #606060;;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="7">
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="m_pSpeaker">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>598</width>
        <height>155</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeaderLabel</class>
   <extends>QLabel</extends>
   <header>headerlabel.hpp</header>
  </customwidget>
  <customwidget>
   <class>NW::Label</class>
   <extends>QLabel</extends>
   <header>nw/label.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/sival.qrc"/>
 </resources>
 <connections/>
</ui>
